frontend/src/user.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  login: {
+    accessToken: null,
+    userId: 0,
+    statusMessage: "",
+  },
+};
+
+export const user = createSlice({
+  name: "user",
+  initialState: initialState,
+  reducers: {
+    setAccessToken: (state, action) => {
+      const { accessToken } = action.payload;
+      console.log(`accessToken: ${accessToken}`);
+      state.login.accessToken = accessToken;
+    },
+
+    setUserId: (state, action) => {
+      const { userId } = action.payload;
+      console.log(`userId: ${userId}`);
+      state.login.userId = userId;
+    },
+
+    setStatusMessage: (state, action) => {
+      const { statusMessage } = action.payload;
+      console.log(`status message: ${statusMessage}`);
+      state.login.statusMessage = statusMessage;
+    },
+
+    logOut: (state, action) => {
+      console.log("logging out");
+      state.login.userId = 0;
+      state.login.accessToken = null;
+    },
+  },
+});
```

Comment:
Reducer is a great place to manage your users but you need to store some of your data in the local storage to keep your users logged in after they refresh the page 🤓